griptape/artifacts/action_chunk_artifact.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import Optional
+
+from attr import define, field
+
+from griptape.artifacts import BaseArtifact, TextArtifact
+from griptape.mixins import SerializableMixin
+
+
+@define
+class ActionChunkArtifact(TextArtifact, SerializableMixin):
```

Comment:
Really don't like this class, but it's required for when we're streaming tool usage. Should we change `ActionsArtifact.Action` into `ActionArtifact`? Then we can yield `ActionArtifact`s much like we yield `TextArtifact`s during streaming.

Reply:
Are you saying that you would yield an `ActionArtifact` after getting all the `ActionChunkArtifact` for that action, instead of yielding the `ActionChunkArtifact`s themselves? (Or would you yield both?)